### PR TITLE
Add Simplified Chinese (China) translation

### PR DESCRIPTION
**Pull request by 5idereal**

### CODE REVIEW

File: TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Strings
+//
+
+STRINGTABLE
+{
+	IDS_WELCOME_NOTIFICATION APP_NAME L" 將隱藏在托盤中。\n\n對托盤圖標點擊右鍵以修改設置、設為開機時啟動、檢視關於資訊、關閉應用...等。"
```
Comment (roxk) [inline TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2]: Still traditional chinese

**Review by roxk — CHANGES_REQUESTED** [review]
Just need 1 correction.

File: TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2 (hunk 1 above)
Comment (5idereal) [reply TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2]: Fixed.

File: Xaml/Strings/zh-CN/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+  </data>
+  <data name="TrayFlyoutPage_VisibleWindow.Text" xml:space="preserve">
+    <value>开启任何窗口时</value>
+  </data>
+  <data name="WelcomePage.Title" xml:space="preserve">
+    <value>欢迎使用 TranslucentTB！</value>
+  </data>
+  <data name="WelcomePage_Action_Configure.Description" xml:space="preserve">
+    <value>您可以随时在托盘图标的右键选单中点击 "高级" &gt; "编辑设置" 来更改
+儲存檔案後，TranslucentTB 將自動載入更動</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: not simplified Chinese

File: Xaml/Strings/zh-CN/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+    <value>关闭</value>
+  </data>
+  <data name="TrayFlyoutPage_About.Text" xml:space="preserve">
+    <value>关于 TranslucentTB</value>
+  </data>
+  <data name="TrayFlyoutPage_AccentColor.Text" xml:space="preserve">
+    <value>主题色</value>
+  </data>
+  <data name="TrayFlyoutPage_Acrylic.Text" xml:space="preserve">
+    <value>压克力</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 压克力 → 亚克力

File: Xaml/Strings/zh-CN/Resources.resw [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+    <comment>Mind the newline (a.k.a. enter)</comment>
+  </data>
+  <data name="WelcomePage_Action_Configure.Name" xml:space="preserve">
+    <value>编辑配置文件</value>
+  </data>
+  <data name="WelcomePage_Action_Discord.Description" xml:space="preserve">
+    <value>与开发者聊聊并取得协助</value>
+  </data>
+  <data name="WelcomePage_Action_Discord.Name" xml:space="preserve">
+    <value>加入我们的 Discord 伺服器</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 伺服器 → 服务器

File: Xaml/Strings/zh-CN/Resources.resw [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+  </data>
+  <data name="WelcomePage_Action_Donate.Description" xml:space="preserve">
+    <value>单次或定期赞助我们以支持开发 ❤</value>
+  </data>
+  <data name="WelcomePage_Action_Donate.Name" xml:space="preserve">
+    <value>透过 Liberapay 赞助此专案</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 透过 → 通过

File: Xaml/Strings/zh-CN/Resources.resw [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+    <value>不透明</value>
+  </data>
+  <data name="TrayFlyoutPage_OpenAtBoot.Text" xml:space="preserve">
+    <value>开机时启动</value>
+  </data>
+  <data name="TrayFlyoutPage_PeekButton.Text" xml:space="preserve">
+    <value>显示桌面预览按钮</value>
+  </data>
+  <data name="TrayFlyoutPage_SearchOpened.Text" xml:space="preserve">
+    <value>开启搜寻菜单时</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 搜寻 → 搜索

File: Xaml/Strings/zh-CN/Resources.resw [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+    <value>隐藏托盘图标</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_LogVerbosity.Text" xml:space="preserve">
+    <value>日志详细级别</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_LogVerbosity_Critical.Text" xml:space="preserve">
+    <value>严重</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_LogVerbosity_Debug.Text" xml:space="preserve">
+    <value>侦错</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 侦错 → 调试

File: TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2 [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Strings
+//
+
+STRINGTABLE
+{
+	IDS_WELCOME_NOTIFICATION APP_NAME L" 将隐藏在托盘中。 \n\n对托盘图标点击右键以修改设置、设为开机时启动、检视关于信息、关闭应用...等。"
+	IDS_HIDETRAY_DIALOG L"此更动仅暂时生效，下次 "APP_NAME L" 启动时将被还原。 \n\n若要永久更动，请透过 「高级」> 「编辑设置」开启配置文件，并将 ""hide_tray"" 设为 false。 \n\n您确定要继续吗？"
```
Comment (TigerCubDen) [inline TranslucentTB/resources/language/TranslucentTB.zh-CN.rc2]: 透过 → 通过

File: Xaml/Strings/zh-CN/Resources.resw (hunk 2 above)
Comment (TigerCubDen) [reply Xaml/Strings/zh-CN/Resources.resw]: 档案 → 文件

File: Xaml/Strings/zh-CN/Resources.resw [hunk 9]
```diff
@@ -0,0 +1,284 @@
...
+    <value>警告</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_OpenLogFile.Text" xml:space="preserve">
+    <value>开启日志文件</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_ResetDynamicState.Text" xml:space="preserve">
+    <value>重置动态状态</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_ResetSettings.Text" xml:space="preserve">
+    <value>恢复出厂设置</value>
```
Comment (TigerCubDen) [inline Xaml/Strings/zh-CN/Resources.resw]: 恢复出厂设置 → 恢复默认设置

**Review by TigerCubDen — COMMENTED** [review]
A little assistance, thanks for your work : )

**Comment by 5idereal** [comment]
All fixed! Thanks for the suggestion, I still got a lot to learn.

**Comment by sylveon** [comment]
Thanks for the translation again! And thank you to @roxk and @TigerCubDen for double checking!

**Pull request merged**